ssc/cmod_generic_system.cpp

```diff
@@ -69,7 +69,6 @@ class cm_generic_system : public compute_module
 		add_var_info( _cm_vtab_generic_system );
 
 		// performance adjustment factors
-		add_var_info(vtab_dc_adjustment_factors);
```

Comment:
Thanks for removing this - that was a great question on DC adjustments for the generic system - maybe DC generators?